services/softlayer_network_storage.go

```diff
@@ -63,7 +76,7 @@ func (slns *softLayer_Network_Storage_Service) CreateIscsiVolume(size int, locat
 	var iscsiStorage datatypes.SoftLayer_Network_Storage
 
 	for i := 0; i < CREATE_ISCSI_VOLUME_MAX_RETRY_TIME; i++ {
-		iscsiStorage, err = slns.findIscsiVolumeIdByOrderId(receipt.OrderId)
+		iscsiStorage, err = slns.findIscsiVolumeId(receipt.OrderId)
 		if err == nil {
 			break
 		} else if i == CREATE_ISCSI_VOLUME_MAX_RETRY_TIME-1 {
```

Comment:
Same again here. We cannot just ignore errors, unless we can do something to recover.